Rakefile

```diff
@@ -16,6 +16,43 @@ Rake::TestTask.new(:test) do |t|
   t.verbose = false
 end
 
+namespace :test do
+  require "tmpdir"
+  require "securerandom"
+
+  desc "Run tests for templates"
+  task :templates => ["templates:all"]
+
+  namespace :templates do
+    task :all => [:npm, :rackup, :mocha, :kill]
+
+    # config
```

Comment:
:scissors:

_(Comments like this don't give much value. As a rule of thumb, I always like to make my comments full sentences and try to say why to code was there to begin with, or something tricky about it. If I can't do that, I don't leave the comment, as it just creates noise to the code.)_